### PR TITLE
logging_*_nodeselector expects json format values

### PR DESCRIPTION
**Pull request by lpsantil**
logging_*_nodeselector expects json format values.  nodeselectors are required by ansible installer playbooks and independent deployer playbooks.  Installer and independent deployer playbook will fail if inputs are not json format.

### CODE REVIEW

**Comment by michaelgugino** [comment]
@lpsantil 

Thank you for the PR.  Just a couple of items.

1)  Please fix against the master branch first, and then back-port to other branches as necessary.

2)  These items should be yaml dictionaries, not json strings.  See the variable 'openshift_builddefaults_nodeselectors' for reference.

File: inventory/byo/hosts.ose.example [hunk 1]
```diff
@@ -641,9 +641,9 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # See https://github.com/openshift/openshift-ansible/tree/release-3.6/roles/openshift_logging
 openshift_logging_es_cluster_size=1
 openshift_logging_es_memory_limit=4G
-openshift_logging_es_nodeselector='purpose=infra'
-openshift_logging_curator_nodeselector='purpose=infra'
-openshift_logging_kibana_nodeselector='purpose=infra'
+openshift_logging_es_nodeselector="{'region':'infra'}"
```
Comment (ewolinetz) [inline inventory/byo/hosts.ose.example]: These should match `openshift_logging_es_nodeselector={'region':'infra'}`, it shouldn't have surrounding double quotes

Comment (lpsantil) [reply inventory/byo/hosts.ose.example]: Thanks

**Comment by lpsantil** [comment]
@michaelgugino Corrected the dictionary syntax.  However `master` branch `byo/host.example` [0] (new name for `byo/host.ose.example`?) does not contain vars for `openshift_logging_*_nodeselector` which is why I submitted a PR only for `release-3.6`.


[0] https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L562-L615

**Comment by michaelgugino** [comment]
@ewolinetz What's going on with these variables in the master branch?  Should we add them?

**Comment by ewolinetz** [comment]
I'm not sure why that isn't in master... I don't see any harm with adding it in there though. 
We do currently have a link to the logging role's readme for further examples for configuration, so that may be why we don't have it in order to keep this example inventory file shorter.

**Comment by michaelgugino** [comment]
Nothing currently in the queue, this is just a docs change.  Merging.

**Pull request merged**